src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {

```

Comment:
That logic makes much more sense now. Is there a meaningful difference in browser between `disabled=""` and `disabled="true"`? If not, I'd opt for the former in both places.